Dockerfile.konflux

```diff
@@ -0,0 +1,53 @@
+FROM registry.redhat.io/ubi9/nodejs-18:latest AS web-builder
+
+WORKDIR /opt/app-root
+
+USER 0
+
+ENV HUSKY=0
```

Comment:
Repo doesn't use HUSKY, so that line can be removed unless we want to add it preemtively and plan to add it later.